front/app/components/PostShowComponents/Comments/Comment/CommentBody.tsx

```diff
@@ -185,10 +184,7 @@ const CommentBody = ({
             setCommentContent('');
           },
           onError: (error) => {
-            if (isCLErrorJSON(error)) {
-              const apiErrors = (error as CLErrorsJSON).json.errors;
-              setApiErrors(apiErrors);
-            }
+            setApiErrors(error.errors);
```

Comment:
In the update comment hook, we used the CLErrorsWrapper right? Does that not mean the errors from updateComment will be wrapped by "`json`" ? :thinking:

Reply:
No, the type does not include `json`. See my comment above.